index.html

```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
         </ul>
     </ul>
   </section>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
+        <li>
+          <input type="text"  name="name" placeholder="Full name" maxlength="30" required/>
+        </li>
+        <li>
+          <input type="email" name="email" placeholder="Email address"  required/>
+        </li>
+        <li>
+          <textarea id="msg" name="user_message" maxlength="500" placeholder="Write me something..." required></textarea>
+        </li>
+        <li>
+		      <button type="submit" class="submit-btn"> Get-in-touch</button>
+        </li>
+        </ul>
+    </form>
+    <ul class="contact-media">
+      <li><a href="#"><img src="./Normal_Button/dark-icon-github.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/dark-icon-linkedin.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/dark-icon-angelist.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/Dark-icon-twitter.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/dark-icon-medium.png"></a></li>
+    </ul>
```

Comment:
- [ ] Please kindly add this shape according to the given template. 👍 

##Figma design
<img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228354721-6bed09c6-582a-4f09-916f-19d1c1b12c06.png">